miden-lib/src/transaction/memory.rs

```diff
@@ -182,14 +182,16 @@ pub const NOTE_MEM_SIZE: MemoryAddress = 512;
 // ------------------------------------------------------------------------------------------------
 // Inputs note section contains data of all notes consumed by a transaction. The section starts at
 // memory offset 1_048_576 with a word containing the total number of input notes and is followed
-// by data of each note like so:
+// by note nullifiers and note data like so:
 //
-//    ┌───────────┬─────────────┬─────────────┬───────────────┬─────────────┐
-//    │ NUM NOTES │ NOTE 0 DATA │ NOTE 1 DATA │      ...      │ NOTE n DATA │
-//    └───────────┴─────────────┴─────────────┴───────────────┴─────────────┘
-// 1_048_576     +1           +513          +1025           +1+512n
+// ┌─────────┬───────────┬───────────┬─────┬───────────┬─────────┬────────┬────────┬─────┬────────┐
+// │   NUM   │  NOTE 0   │  NOTE 1   │ ... │  NOTE n   │ PADDING │ NOTE 0 │ NOTE 1 │ ... │ NOTE n │
+// │  NOTES  │ NULLIFIER │ NULLIFIER │     │ NULLIFIER │         │  DATA  │  DATA  │     │  DATA  │
+// └─────────┴───────────┴───────────┴─────┴───────────┴─────────┴────────┴────────┴─────┴────────┘
+//  1_048_576  1_048_577   1_048_578        1_048_576+n      1_064_960   +512    +1024  +512n
 //
-// Data section for each note consists of exactly 512 words and is laid out like so:
+// Each nullifier occupies a single word the data section for each note consists of exactly 512
```

Comment:
Is there a period or a comma missing?

`Each nullifier occupies a single word the data section for each note consists of exactly 512`

Reply:
Fixed!